src/databricks/labs/ucx/runtime.py

```diff
@@ -434,11 +435,19 @@ def migrate_external_tables_sync(
     table_crawler = TablesCrawler(sql_backend, cfg.inventory_database)
     udf_crawler = UdfsCrawler(sql_backend, cfg.inventory_database)
     grant_crawler = GrantsCrawler(table_crawler, udf_crawler)
-    table_mapping = TableMapping(install, ws, sql_backend)
+    table_mappings = TableMapping(install, ws, sql_backend)
```

Comment:
Don't rename

Reply:
i am getting this error in linter if I don't rename 
************* Module tests.performance.test_performance
tests/performance/test_performance.py:1:0: R0801: Similar lines in 2 files
==databricks.labs.ucx.runtime:[470:477]
==tests.integration.hive_metastore.test_migrate:[145:152]
        table_crawler,
        grant_crawler,
        ws,
        sql_backend,
        table_mapping,
        group_manager,
        migration_status_refresher, (duplicate-code)
tests/performance/test_performance.py:1:0: R0801: Similar lines in 2 files
==databricks.labs.ucx.runtime:[442:449]
==tests.integration.hive_metastore.test_migrate:[76:83]
        table_crawler,
        grant_crawler,
        ws,
        sql_backend,
        table_mapping,
        group_manager,
        migration_status_refresher, (duplicate-code)